cypress/fixtures/upgrade-data.json

```diff
@@ -11,5 +11,6 @@
     "binaryApplicationName": "upgrade-binaryApp",
     "uploadBinaryApplicationName": "upgrade-uploadBinaryApp",
     "assessmentApplicationName": "upgrade-assessmentApplication",
-    "archetypeName": "upgrade-archetype"
+    "archetypeName": "upgrade-archetype",
+    "importedQuestionnaireAppName": "upgrade-assessmentImport"
```

Comment:
Let's rename this to "upgrade-questionnaireApplication"

Reply:
Done in https://github.com/konveyor/tackle-ui-tests/pull/1094/commits/bc06cd9642ef79e162e7399ec6316aaf1cfa882e